packages/kit/src/exports/vite/index.js

```diff
@@ -502,6 +518,134 @@ async function kit({ svelte_config }) {
 						}
 					`;
 				}
+
+				// The virtual module that is imported in the environment entrypoint files. This provides all the data that is needed to create the `Server` instance.
+				// Not implemented:
+				// - Server assets. The `read` function from `$app/server` can only be used in environments that support file system access.
+				// - Inlining styles. This requires communicating with the main process to collect dependencies. It should be possible (e.g. using import.meta.hot) but needs more investigation.
+				case sveltekit_environment_context: {
+					const { manifest_data, env, remote_address } = environment_context;
+
+					return dedent`
+						import { create_resolve } from "${runtime_base}/server/environment_context.js";
+
+						const resolve = create_resolve(${s(cwd)});
+
+						export let manifest = {
+							appDir: ${s(svelte_config.kit.appDir)},
+							appPath: ${s(svelte_config.kit.appDir)},
+							assets: new Set(${s(manifest_data.assets.map((asset) => asset.file))}),
+							mimeTypes: ${s(get_mime_lookup(manifest_data))},
+							_: {
+								client: {
+									start: "${runtime_base}/client/entry.js",
+									app: "${to_fs(svelte_config.kit.outDir)}/generated/client/app.js",
+									imports: [],
+									stylesheets: [],
+									fonts: [],
+									uses_env_dynamic_public: true
+								},
+								server_assets: {},
+								nodes: [
+									${manifest_data.nodes
+										.map((node, i) => {
+											const index = s(i);
+											const component = s(node.component);
+											const universal = s(node.universal);
+											const server = s(node.server);
+
+											return dedent`
+												async () => {
+													const result = {};
+
+													const module_nodes = [];
+
+													result.index = ${index};
+
+													// these are unused in dev, it's easier to include them
+													result.imports = [];
+													result.stylesheets = [];
+													result.fonts = [];
+
+													if (${component}) {
+														result.component = async () => {
+															const { module } = await resolve(${component});
```

Comment:
Missed the return statement here:

```suggestion
															const { module } = await resolve(${component});
															
															return module.default;
```